examples/island_cluster_with_vm_router/README.md

```diff
@@ -0,0 +1,87 @@
+# GKE island cluster using VM as router
+
+This example provisions a cluster in an island VPC allowing reuse of the IP address space for multiple clusters in the same project. 
+
+1. An appliance(VM as router) with multiple NICs is used to establish connectivity between the island VPC and the existing network.
+2. Outbound connections will go through the router.
+3. For inbound connections, use Private Service Connect.
+
+## Deploy
+
+1. Use the following example to deploy a cluster using this module.
+2. Update `project_id` and `primary_subnet` values in `terraform.tfvars`, and update other variables as needed.
+3. Value for `random_suffix` variable needs to be provided for a successful terraform plan.
```

Comment:
Can we use the random resource to generate one for the user?

Reply:
Ran into `tf` couldn't plan since some resources are known after apply, changed this to use `cluster_name` prefix and updated readme.